driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -54,6 +52,22 @@
 
     private static final String CASSANDRA_VERSION_REGEXP = "\\d\\.\\d\\.\\d+(-\\w+)?";
 
+    public static final String DEFAULT_CLIENT_TRUSTSTORE_PASSWORD = "cassandra1sfun";
+    public static final String DEFAULT_CLIENT_TRUSTSTORE_PATH = "/client.truststore";
+
+    public static final String DEFAULT_CLIENT_KEYSTORE_PASSWORD = "cassandra1sfun";
+    public static final String DEFAULT_CLIENT_KEYSTORE_PATH = "/client.keystore";
+
+    public static final String DEFAULT_SERVER_TRUSTSTORE_PASSWORD = "cassandra1sfun";
+    public static final String DEFAULT_SERVER_TRUSTSTORE_PATH = "/server.truststore";
+
+    private static final File DEFAULT_SERVER_TRUSTSTORE_FILE = createTempStore(DEFAULT_SERVER_TRUSTSTORE_PATH);
```

Comment:
The reason is for portability,  since we build the test jar and it is made available to other modules they can't access the keystores by the same relative path.   However, I kind of agree that this is complicated and unnecessary in the general case (running tests from driver-core), might be better to just use relative path to the src/test/resources folder instead for the time being.  What do you think?